### PR TITLE
add pyshacl (and owlrl)

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
  - some patches required due to misbehaved old-school `scripts` in `setup.py`
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by bollwyvl** [comment]
@satra any interest in co-maintaining?

**Comment by bollwyvl** [comment]
Grr... looks like `bin/pyshacl` will take some patching...

**Comment by bollwyvl** [comment]
@satra thanks for the :+1: ! Please drop an actual comment (can also just be a :+1:, though), as emoji don't leave much of a formal trail...

**Comment by satra** [comment]
@bollwyvl - happy to help co-maintain.

**Comment by satra** [comment]
you can also add @sanuann from our group who is using pyshacl for a project currently

**Comment by bollwyvl** [comment]
@satra, cool: let's get @sanuann to also add a personal comment, but that shouldn't block review, now that we're :green_heart:  (albeit with some patching).

@conda-forge/help-python ready for a look! I'm working on upstream PRs to avoid the patches in potential future releases.

**Comment by bollwyvl** [comment]
Upstream PR up for `entry_point` change:
https://github.com/RDFLib/pySHACL/pull/23

File: recipes/owlrl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - owlrl
+  commands:
+    - owlrl --help
+    - cd src && cd tests && pytest --cov=owlrl -k "not test_dt_not_type" -vv
+
+about:
+  home: https://github.com/RDFLib/OWL-RL
+  license: W3C
+  license_family: OTHER
+  license_file: {{ environ["RECIPE_DIR"] }}/w3c.txt
```
Comment (chrisburr) [inline recipes/owlrl/meta.yaml]: It will automatically fall back to looking there:
```suggestion
  license_file: w3c.txt
```

Comment (bollwyvl) [reply recipes/owlrl/meta.yaml]: So convenient!

**Comment by bollwyvl** [comment]
@chrisburr don't know why accepting your commit caused appveyor to fail... added some more meta to trigger a build... seems to be working!

**Comment by bollwyvl** [comment]
Hooray!

**Comment by satra** [comment]
nice!

**Pull request merged**